### PR TITLE
Added README.md

### PR DESCRIPTION
**Pull request by Nivedita967**
Adding a README will help people interested in this repository to understand your project (easily and smoothly).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Trebleshot
+<b>A SHARING APP FOR MOBILE, DESKTOP & BROWSERS.</b><br>
+An Open-Source Android application that allows you to send and receive files over available connections, and offers unique features like sharing over HTTP, pausing and resuming transfers.<br>
+Download it today!<br>
```
Comment (velitasali) [inline README.md:4]: Using two lines, in other words `\n\n`, will have the same effect as using <br>

File: README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+<a href="https://play.google.com/store/apps/details?id=com.genonbeta.TrebleShot">Play Store</a><br> <a href="https://github.com/trebleshot/android">GitHub</a><br><br>
+<b>It's main features-</b><br>
+1> Resume Transfers-<br>
```
Comment (velitasali) [inline README.md:7]: There is a better formatting option for numbered bullet points:

```markdown
1. First
2. Second
3. Third
```

**Review by velitasali — CHANGES_REQUESTED** [review]
Thanks for working on this. Other than the formatting and small coherence issues, this is mergeable.

**Comment by velitasali** [comment]
Alright, this looks good for a start. Other people can improve upon it. 

Thanks :+1:

**Comment by Nivedita967** [comment]
Sure.

**Pull request merged**